framework/src/modules/chain/synchronizer/utils.js

```diff
@@ -51,10 +51,15 @@ const restoreBlocks = async (blocksModule, processorModule, tx) => {
  * @return {Promise<void>} - Promise is resolved when blocks are successfully deleted
  */
 const deleteBlocksAfterHeightAndBackup = async (
+	logger,
 	processorModule,
 	blocksModule,
 	desiredHeight,
 ) => {
+	logger.debug(
+		{ height: desiredHeight },
+		'Deleting blocks after target height',
+	);
```

Comment:
would it make sense to add another logger.debug message after the while loop telling something like `blocks deleted successfully`?

Reply:
Agreed